image_upgrade/image_upgrade/docker_hub.py

```diff
@@ -0,0 +1,60 @@
+import httpx
```

Comment:
```suggestion
# Copyright (C) 2024 Greenbone Networks GmbH
#
# SPDX-License-Identifier: GPL-3.0-or-later
#
import httpx
```

Reply:
should be
```
# SPDX-FileCopyrightText: 2024 Greenbone AG
```
btw.

Reply:
You are correct.
But @pascalholthaus can run `pontos-update-header` when he is ready.